### DISCUSSION TITLE
Copy individual cell values from Table

### DISCUSSION THREAD

**Discussion by LenaRampula** [h2oai/wave#777] [Q&A]
I would like to utilize the search capability of the table component to search for relevant information and allow users to copy a value from a cell.
As the cell contains long text it is very difficult to copy the entire text even if the column is expanded.

What is the best way to implement this? Is there a different way to accomplish this?

**Comment by mturoci** [comment]
@shanaka-rajitha any UX suggestion for this enhancement?
